es6.md

```diff
@@ -75,9 +82,16 @@ A `yield` expression.
 ## ObjectPattern
 
 ```js
+interface PatternProperty <: Property {
```

Comment:
@fkling Named it so because in this case it's not "property pattern" as it's not pattern itself, but rather "pattern's property". However exact name does not matter here since it doesn't affect `type` property and is used only for internal needs.